src/index.ts

```diff
@@ -21,6 +28,7 @@ export default class BlueButton {
   callbackUrl: string;
   version: string;
   baseUrl: string;
+  private _authResponseStatusCode: number;
```

Comment:
We shouldn't store variables like this on the BB class. This should just hold the high level config for the app to make requests to BB2. It shouldn't be storing anything on any individual auth flow/fhir request. A single class could be authorizing a lot of beneficiaries, fetching a lot of data.

Reply:
@nbragdon I removed this.